ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -606,14 +606,17 @@ def _generate_physical_disks(self):
 
     @mock.patch.object(drac_common, 'get_drac_client', spec_set=True,
                        autospec=True)
+    @mock.patch.object(drac_common, 'get_reboot_options', spec_set=True,
+                       autospec=True)    
```

Comment:
I think there's some trailing whitespace on line 610 that needs to be removed.

Reply:
Done.